<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1068</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="error">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>35</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>10</y>
      <width>169</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="printClearLayout">
     <item>
      <widget class="QPushButton" name="printButton">
       <property name="text">
        <string>Print</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clrButton">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>921</width>
      <height>451</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>902</width>
       <height>432</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>843</x>
      <y>10</y>
      <width>154</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="maxDegOptLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Max Deg.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="maxDegOpt">
       <property name="maxVisibleItems">
        <number>4</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>508</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="userInputLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Enter a number</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="textbox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Enter a integer between 0 and 9999</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="searchButton">
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="insButton">
       <property name="text">
        <string>Insert</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="revButton">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="history">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>560</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&gt;&gt;&gt; History appear here</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1068</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
